### PR TITLE
Fall back to base locale if a countries specific locale is not available

### PR DESCRIPTION
**Pull request by amenk**
i.e. de-CH -> de

### CODE REVIEW

File: src/FileLoader.php [hunk 1]
```diff
@@ -53,4 +53,26 @@ public function load($locale, $group, $namespace = null)
 
         return array_replace_recursive($defaults, parent::load($locale, $group, $namespace));
     }
+    
+        /**
```
Comment (overtrue) [inline src/FileLoader.php]: error intend

File: src/FileLoader.php [hunk 2]
```diff
@@ -53,4 +53,26 @@ public function load($locale, $group, $namespace = null)
...
+     * Fall back to base locale (i.e. de) if a countries specific locale (i.e. de-CH) is not available
+     *
+     * @param string $path
+     * @param string $locale
+     * @param string $group
+     * @return array|mixed
```
Comment (overtrue) [inline src/FileLoader.php]: empty line before `@return`

File: src/FileLoader.php [hunk 3]
```diff
@@ -53,4 +53,26 @@ public function load($locale, $group, $namespace = null)
...
+    {
+        $result = parent::loadPath($path, $locale, $group);
+
+        if ($result == [] && strpos($locale, '-') !== false) {
+            list($baseLocale) = explode('-', $locale);
+            if ($this->files->exists($full = "{$path}/{$baseLocale}/{$group}.php")) {
+                return $this->files->getRequire($full);
+            }
+        }
+
```
Comment (overtrue) [inline src/FileLoader.php]:
```php
if (empty($result) && str_contains($locale, '-')) {
    return parent::loadPath($path, strstr($locale, '-', true), $group);
}
```

Comment (overtrue) [inline src/FileLoader.php]: maybe you can simplify it more. 😄

**Comment by amenk** [comment]
thanks for the changes, if you want you can commit all of them :-)

​

**Pull request merged**